.changeset/honest-dragons-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-llama": patch
+---
+
+Add folder selection and support more file types
```

Comment:
does it work with express, fastapi and nextjs?

Reply:
yes, it is